libnavigation-core/src/main/java/com/mapbox/navigation/core/telemetry/events/CoreTelemetryEventUtils.kt

```diff
@@ -0,0 +1,66 @@
+package com.mapbox.navigation.core.telemetry.events
+
+import com.mapbox.bindgen.Value
+
+internal fun String.toValue(): Value = Value.valueOf(this)
+internal fun Boolean.toValue(): Value = Value.valueOf(this)
+internal fun Int.toValue(): Value = Value.valueOf(this.toString().toLong())
+internal fun Double.toValue(): Value = Value.valueOf(this)
+internal fun Float.toValue(): Value = Value.valueOf(this.toString().toDouble())
```

Comment:
without it cast is not accurate: 
```kotlin
@Test
fun testCast() {
    assertNotEquals(1.1, 1.1f.toDouble(), 0.0)

    assertEquals(1.1, 1.1f.toString().toDouble(), 0.0)
}
```

Reply:
seems we can guarantee accuracy for free, let's do it

Reply:
But can we really rely on toString and converting back from it? I mean is it guaranteed that once it is converted to String it can be then restored? Example: 18768768.768987398f is converted to "1.8768768E7". Can it be restored from that format? Surely yes, but is it true for all cases?

Reply:
discussed internally: 
- keep `Float.toDouble()`
- cut a ticket for common SDK to support Value.ValueOf(Float)

Reply:
cut `#CORESDK-1351`